site/content/docs/5.0/guidelines/global-headers.md

```diff
@@ -7,6 +7,19 @@ subgroup: Modules
 toc: true
 ---
 
-{{< callout info >}}
-The feature will be delivered with [#525]({{< param repo >}}/issues/525).
-{{< /callout >}}
+<div class="container">
+  <h2 id="suprabar">Supra bar</h2>
+  <p>
+    <strong>Supra bars are hidden on tablet and mobile devices.</strong>
+    Based on viewport size, supra bar is hidden under <var>768px</var>.
+  </p>
+  <h3 class="mt-5 h5">Supra bar standard — <a id="web-glh-sub-001" class="ui-kit-id">web-glh-sub-001</a></h3>
+</div>
+<nav class="navbar navbar-dark bg-dark navbar-expand-md pt-0 d-none d-sm-block d-sm-none d-md-block">
+  <div class="container">
+      <div class="navbar-nav">
+        <a href="#" class="nav-link active pb-1" aria-current="page">Personal</a>
+        <a class="nav-link pb-1" href="#">Business</a>
```

Comment:
Just a detail 💅 but it would be easier to read to have `href` and `class` in the same order between those 2 `<a>`s.